nishimweelysee/Elysee-CapstoneProject
# Ft user about me page

#### What does this PR do?
> This PR Shows the website owner Profile  profile information, picture, skills, experience ......
>
#### Description of Task to be completed
>- Profile Image
>- Experience
>- Physical Address
>- Skills
>- Development Languages
>
#### How should this be manually tested?
> Open website and navigate to About me page you will see all information about me.
#### Any background context you want to provide?
>
#### What are the relevant Trello story?
> Ft(User About me Page) use this link https://trello.com/c/098yavmz to visit my trello board
#### Screenshots (if appropriate)
>
![image](https://user-images.githubusercontent.com/56052588/92327330-1b5c2280-f059-11ea-86b3-c69e783d8134.png)

![image](https://user-images.githubusercontent.com/56052588/92327338-2ca52f00-f059-11ea-92a2-0ec74ee5b3e5.png)
